template_memo.html

```diff
@@ -0,0 +1,337 @@
+					float: none !important;
+					width: 100% !important;
+				}
+				.sidebox-padding {
+					padding-left: 0px !important;
+				}
+			}
+			/* DARK MODE FOR IOS AND APPLE MAIL */
+			@media (prefers-color-scheme: dark) {
+				body {
+					/* background: #1c1c1e; */
+				}
+				a,
+				a:visited,
+				a:hover,
+				a:active {
+					color: #2d86e0;
+				}
+				a.button-text,
+				a.button-text:visited,
+				a.button-text:hover,
+				a.button-text:active {
+					color: #cecece;
+				}
+				td.emailbodytext p,
+				td.emailbodytext li,
+				td.emailbodytext h1,
+				td.emailbodytext h2,
+				td.emailbodytext h3,
+				td.tagline p,
+				td.dm-box h2 {
+					color: #cecece;
+				}
+				td.dm-box {
+					background: #2a2a2b !important;
+					color: #cecece;
+				}
+				td.emailbodytext {
+					/* border-bottom: 1px solid #2a2a2b !important; */
+				}
+				table.inner-sidebox,
+				td.dm-box {
+					border: 0 !important;
+				}
+			}
+		</style>
+	</head>
+	<body>
+		{include "dataLoader"}
+
+		<!-- <span class="preheader" style="display: none !important; visibility: hidden; opacity: 0; color: transparent; height: 0; width: 0">ENTER PREHEADER TEXT MESSAGE.</span>{include "pre-header-spacing"} --> {beacon}
+
+		<table role="presentation" align="center" border="0" cellpadding="0" cellspacing="0" class="emailwrapto100pc" width="600px">
+			<tbody>
+				<!-- <tr>
+					<td align="center" style="padding: 20px 0">
+						<a href="{website_url}"><img alt="{affiliation_name}'s logo" src="{logo_url}" border="0" width="{logo_width}" style="display: block" /></a>
+					</td>
+				</tr> -->
+				<tr>
+					<td class="emailbodytext" style="color: #3c3532;font-family: Arial,&#39;Helvetica Neue&#39;,sans-serif;font-size: 16px;line-height: 22px;padding: 0; border-bottom: 1px solid #d4d4d4;" valign="top">
+						<!-- HERO IMAGE AND CTA IMAGE TABLE -->
+						<!-- IMPORTANT: REFORMAT IMAGE TO 600px OR IT WILL BREAK IN OUTLOOK -->
+						<table role="presentation" cellpadding="0" cellspacing="0" border="0" width="600px">
+							<tr>
+								<td style="padding: 0;">
+									<table role="presentation" cellpadding="0" cellspacing="0" border="0" width="600px" class="emailwrapto100pc">
+										<tbody>
+											<tr>
+												<td>
+													<a href="{website_url}"><img src="https://media.sailthru.com/64d/1k8/5/s/66561f02cfa77.jpg" width="600px" class="image-resize" border="0" alt="Freedom can't protect itself text with the Statue of Liberty's torch." style="display: block" /></a>
+												</td>
+											</tr>
+										</tbody>
+									</table>
+								</td>
+							</tr>
+						</table>
+						<!-- HERO IMAGE AND CTA IMAGE TABLE -->
+
+                        <table role="presentation" width="600px" cellspacing="0" cellpadding="0" border="0">
+                        <tr>
+                            <td style="padding: 40px 60px; background: #DCD3C3;">
+
+								<table role="presentation" width="100%" cellspacing="0" cellpadding="0" border="0">
```

Comment:
Can you make sure this is all in a single line? Looks like it is breaking up into multiple lines?